src/support-vuejs/index.md

```diff
@@ -1,25 +1,25 @@
-# Support Vue.js Development
+# Soutenir le développement de Vue.js
 
-Vue.js is an MIT licensed open source project and completely free to use.
-However, the amount of effort needed to maintain and develop new features for the project is not sustainable without proper financial backing. You can support Vue.js development via the following methods:
+Vue.js est un projet open source sous licence MIT qui est complètement libre d'utilisation.
+Cependant, les efforts nécessaires à la maintenance et au développement de nouvelles fonctionnalités pour le projet ne sont pas possibles sans un soutien financier approprié. Vous pouvez soutenir les développements Vue.js des manières suivantes :
 
-- [Become a backer or sponsor via Patreon](https://www.patreon.com/evanyou) (goes directly to support Evan You's full-time work on Vue)
+- [Devenir un donateur ou un sponsor via Patreon](https://www.patreon.com/evanyou) (offre un support direct à Evan You pour un travail à temps complet sur Vue)
```

Comment:
(offre un support direct ~~à~~ **vers** Evan You **qui travaille** ~~pour un travail~~ à temps complet sur Vue)

Reply:
je garderais à, "offrir vers" ne veut pas dire grand chose

Reply:
@sylvainpolletvillard 

Je comprends la phrase comme quoi si on devient donateur, en échange on a un support qui est offert. Et ce support est fait par Evan You qui travaille à temps complet.

Donc "offrir à" me gène, ce n'est pas **à** Evan You qu'on offre, mais plutôt au donateur

Il faut peut-être changer la tournure de la phrase. Qu'en penses tu ?

Reply:
Non, ce n'est pas le sens de la phrase. Evan You a déjà précisé sur son Patreon qu'il n'offrait pas de meilleur support aux donateurs, c'était d'ailleurs un point clé dans sa campagne. Don != support premium

Ici une traduction alternative serait "sert directement à supporter le travail à plein temps d'Evan You sur Vue"

Reply:
Ok. merci pour cette précision. 
Je préfère personnellement ta proposition alternative.

Reply:
C'est dans la boite